test/test_ngram.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+
+from math import isclose
+
+kenlm = pytest.importorskip("kenlm")
+
+test_sens = ["I like apple", "you love coffee"]
+
+
+@pytest.mark.parametrize("test_sens", [test_sens])
+def test_ngram_build(test_sens):
+    lm = kenlm.LanguageModel("test/test.arpa")
```

Comment:
`test/test.arpa` supposes the executing directory level.

The following file is a good example.
https://github.com/espnet/espnet/blob/master/test/test_sentencepiece.py

Reply:
done